src/app/pages/tx-summary/tx-summary.tsx

```diff
@@ -0,0 +1,62 @@
+import { FiCheck } from 'react-icons/fi';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { Box, Flex } from '@stacks/ui';
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { useRouteHeader } from '@app/common/hooks/use-route-header';
+import { Header } from '@app/components/header';
+
+import { TxDetails, TxSummaryDetails } from './tx-summary-details';
+import { TxSummaryFooter } from './tx-summary-footer';
+import { TxSummaryValue } from './tx-summary-value';
+
+export function TxSummary() {
+  const navigate = useNavigate();
+  const { state } = useLocation();
+
+  console.log('tx', state);
+
+  useRouteHeader(
+    <Header hideActions onClose={() => navigate('..', { relative: 'path' })} title="Sent" />
+  );
+
+  // TO-DO load tx by id if state is empty
+  if (!state.transaction) return null;
+  const { transaction, txValue, txFiatValue, symbol, txLink, analyticsId, fastestFee } = state;
+  const details: TxDetails[] = [
+    { title: 'To', value: truncateMiddle(transaction.vout[0].scriptpubkey_address, 4) },
+    { title: 'Arrives in', value: fastestFee },
+    'line',
+    {
+      title: 'Total amount',
+      value: txValue,
+    },
+
+    { title: 'Sending', value: txValue },
+    {
+      title: 'On-chain fee',
+      value: transaction.fee,
+    },
+    'line',
+    {
+      title: 'Resulting balance',
+      value: txValue,
+    },
+  ];
+  return (
+    <Flex
+      flexDirection="column"
+      pt="extra-loose"
+      px="extra-loose"
+      alignItems="center"
+      justifyItems="centers"
+      width="100%"
+    >
+      <Box as={FiCheck} size="40px" />
+      <TxSummaryValue value={txValue} fiatValue={txFiatValue} symbol={symbol} />
+      <TxSummaryDetails details={details} />
+      <TxSummaryFooter id={transaction.txId} link={txLink} analyticsId={analyticsId} />
```

Comment:
I don't see any need to pass analytics id through the state versus just adding it directly here? Does it need to be dynamic?

Reply:
I think the better way to do this is to have a fixed event, and alternate the data with event details:

```
void analytics.track('view_transaction', { symbol: 'BTC' });
```

cc @markmhx

Reply:
SGTM, would just recommend event name `view_transaction_confirmation` to specify further